apple/internal/apple_framework_import.bzl

```diff
@@ -318,11 +318,36 @@ def _debug_info_binaries(
 
     return all_binaries_dict.values()
 
+def _get_framework_imports(ctx):
+    if not ctx.attr.xcframework_platform_ids:
+        return ctx.files.framework_imports
+    curr_platform = ctx.fragments.apple.single_arch_platform.name_in_plist.lower()
```

Comment:
@thii Because of the known bug with `ctx.fragments.apple.single_arch_platform`, we should patch this similar to https://github.com/bazelbuild/rules_swift/pull/694, after this is merged. Then we can revert both patches when Bazel core is fixed.

Reply:
The bug is with `single_arch_cpu`, not `single_arch_platform`.